pages/linux/distrobox.md

```diff
@@ -0,0 +1,41 @@
+# distrobox
+
+> Use any Linux distribution inside your terminal by running it as a container.
+> Created container will be tightly integrated with the host, allowing sharing of the HOME directory of the user, external storage, external USB devices and graphical apps (X11/Wayland), and audio.
+> Uses `podman` or `docker` underneath.
```

Comment:
Up to 2 command description lines, and just 1 more information link is allowed.

Reply:
Okay, I'll modify it.